src/components/IDO/IDO.tsx

```diff
@@ -0,0 +1,120 @@
+import React, { useEffect, useState } from 'react'
+import { Grid, Typography, CardMedia } from '@material-ui/core'
+import AmountInput from '@components/Inputs/ExchangeAmountInput/ExchangeAmountInput'
+import { OutlinedButton } from '@components/OutlinedButton/OutlinedButton'
+import AccessTimeIcon from '@material-ui/icons/AccessTime'
+import icons from '@static/icons'
+import { useStyles } from './style'
+
+export const IDO = () => {
+  const classes = useStyles()
+  const [est, setEst] = useState<string>(' ')
+  const [balance, setBalance] = useState<string>(' ')
+  const [inputValue, setInputValue] = useState<string | undefined>(' ')
+  const [depositedXUSD, setDepositedXUSD] = useState<string>(' ')
+  const [depositedUSD, setDepositedUSD] = useState<string>(' ')
+  const [depositedSOL, setDepositedSOL] = useState<string>(' ')
+  const [depositedXETH, setDepositedXETH] = useState<string>(' ')
+  const [depositedXBTC, setDepositedXBTC] = useState<string>(' ')
+  const [salePeriod, setSalePeriod] = useState<string>(' ')
+  const [gracePeriod, setGracePeriod] = useState<string>(' ')
+  const [SOLContributed, setSOLContributed] = useState<string>(' ')
+  const [estTokenPrice, setEstTokenPrice] = useState<string>(' ')
+  const [INVForSale, setINVForSale] = useState<string>(' ')
+
+  useEffect(() => {
+    setEst('56.0278')
```

Comment:
mock data should be set through props in story, not be hardcoded inside component

Reply:
mock data i handler functions zakodowane w komponencie są tylko na potrzeby zadania, żeby odwzorować makietę